src/pages/permissions/actions/LxdGroupActions.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from "react";
+import { ContextualMenu } from "@canonical/react-components";
+import { LxdGroup } from "types/permissions";
+import EditLxdGroupAction from "./EditLxdGroupAction";
+import DeleteLxdGroupAction from "./DeleteLxdGroupAction";
+import EditLxdGroupIdentitiesAction from "./EditLxdGroupIdentitiesAction";
+import EditLxdGroupEntitlementAction from "./EditLxdGroupEntitlementAction";
+interface Props {
+  group: LxdGroup;
+}
+
+const LxdGroupActions: FC<Props> = ({ group }) => {
+  return (
+    <ContextualMenu
+      dropdownProps={{ "aria-label": "actions menu" }}
+      position="right"
+      toggleClassName="u-no-margin--bottom u-no-padding--top u-no-padding--bottom"
+      toggleProps={{
+        "aria-label": "actions selection toggle",
+      }}
+      toggleLabel={
+        <img
+          src="/ui/assets/icon/contextual-menu.svg"
+          alt="lxd group actions menu toggle"
+        />
```

Comment:
@piperdeck and I had a discussion about this. We agreed since this is a general trend that all other apps will adopt, we will implement this for lxd as well. Starting with permissions first then later on we will apply it across all other tables with actions. Although for a single action I think we will still have it directly in the table without a contextual menu.

Reply:
Personally, I believe having the buttons directly available is the better approach - at least if we have clear iconography and there are not too many options.